mon-pix/app/controllers/application.js

```diff
@@ -4,5 +4,5 @@ import Controller from '@ember/controller';
 export default class ApplicationController extends Controller {
   @service intl;
 
-  pageTitle = this.intl.t('page-title.suffix');
+  pageTitle = this.intl.t('navigation.suffix');
```

Comment:
Je trouve important de faire apparaître la notion de page-title.
Pourquoi est-ce qu'on n'en a plus besoin ?
```suggestion
  pageTitle = this.intl.t('navigation.page-title.suffix');
```

Reply:
Il y a effectivement (suite aux commentaires précédent) un mélange entre Pix le titre, et Pix la "page" en quelque sorte. Autant les séparer dans le fichier.